concourse/scripts/common.bash

```diff
@@ -87,6 +87,11 @@ function make_cluster() {
 	source /usr/local/greenplum-db-devel/greenplum_path.sh
 	export BLDWRAP_POSTGRES_CONF_ADDONS=${BLDWRAP_POSTGRES_CONF_ADDONS}
 	export STATEMENT_MEM=250MB
+
+    if [[ "$MAKE_TEST_COMMAND" =~ "optimizer=on" ]]; then
```

Comment:
The indentation here (spaces) differs from the indentation around (tabs). Use tabs